posts/inside-rust/2025-02-21-relnotes-interest-group.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Relnotes PR and release blog post ping group"
+author: Jieyou Xu
+team: The Release Team <https://www.rust-lang.org/governance/teams/infra#team-release>
```

Comment:
No idea what `team` field to use here because I'm obviously not part of release

Reply:
Seems fine.